engine/resource/src/resource_archive.cpp

```diff
@@ -593,12 +581,6 @@ namespace dmResourceArchive
             {
                 fclose(afi->m_FileResourceData);
             }
-
```

Comment:
The actual bug: We didn't let the Unmount-function handle this, and instead incorrectly assumed it should be released using "UnmapFile".